src/styles/blocks/header.scss

```diff
@@ -0,0 +1,50 @@
+.container {
```

Comment:
Consider moving the .container class to a separate file, like '_layout.scss' or '_container.scss', to keep the file focused on the header block.